plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/AssociationPart.java

```diff
@@ -104,6 +106,18 @@ protected void createEditPolicies() {
 
     @Override
     protected IFigure createFigure() {
+        DBRProgressMonitor monitor = new VoidProgressMonitor();
```

Comment:
You're still creating VoidProgressMonitor here instead of using real monitor from job